src/makeCalculator.js

```diff
@@ -38,6 +38,29 @@
  */
 function makeCalculator() {
   // write code here
+  let result = {
+    result: 0,
+    add: (value) => {
+      this.result += value;
+    },
```

Comment:
You dont need to use ':' and '=>'
add(value) { this.result += value; } - it works the same.